changelog.d/9517.doc

```diff
@@ -0,0 +1 @@
+Complete instructions for Apache proxying
```

Comment:
You will need to add punctuation (just a period) to the end of this changelog entry.